### PR TITLE
UGIC Recap Blogpost

### PR DESCRIPTION
**Pull request by jpechmann**
Pull request description
------------------------

### CODE REVIEW

**Comment by jpechmann** [comment]
Oh, and I edited the parcel post a bit too.

File: _posts/2016-05-26-agrc-s-latest-where-in-utah-quiz-and-ugic-2016-conf-recap.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+categories:
+
+tags:
+- UGIC
+- Conference
+- Quiz
+- AGRC
+- Bryce
+- Utah
+
```
Comment (steveoh) [inline _posts/2016-05-26-agrc-s-latest-where-in-utah-quiz-and-ugic-2016-conf-recap.md]: try not to have an empty line here

File: _posts/2016-05-26-agrc-s-latest-where-in-utah-quiz-and-ugic-2016-conf-recap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+status: publish
+published: true
+title: 'AGRC's Latest Where in Utah Quiz and UGIC 2016 Conf Recap'
```
Comment (steveoh) [inline _posts/2016-05-26-agrc-s-latest-where-in-utah-quiz-and-ugic-2016-conf-recap.md]: you can't have a single quote in the title

**Comment by jpechmann** [comment]
@steveoh  Okay, I fixed it.

**Pull request closed without merging**